README.md

```diff
@@ -1,218 +1,9 @@
 # Doppler HTML Editor API
 
-## API preliminar design
+This API allows to store and retrieve Doppler campaign content from Doppler DB.
 
-### Campaigns
+At the moment, it only support HTML content (Import or HTML Editor) and Unlayer Content. In the future we could also deal with other contents like MSEditor.
 
-```http
-GET /accounts/{accountName}/campaigns/{campaignId}/content
-```
+## Project Architecture
```

Comment:
@Cromeror @maurocardinali @GusBaamonde I have updated the documentation. 

I think that the API examples were outdated, so I removed them (we have the real API now)

You can easily review it with rich diff: 

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/1157864/161328591-b2bc1c8b-806b-4083-bfc6-5f7145235c04.png">